mock/py/mockbuild/plugins/chroot_scan.py

```diff
@@ -56,7 +56,7 @@ def __scanChroot(self):
         regexstr = "|".join(self.scan_opts['regexes'])
         regex = re.compile(regexstr)
         chroot = self.buildroot.make_chroot_path()
-        file_util.mkdirIfAbsent(self.resultdir)
+        self.buildroot.create_resultdir()
```

Comment:
Argh! While it _was_ causing issues before, now it is broken as well -> because `self.resultdir` is not `serlf.buildroot.resultdir`, nah.  chroot_scan is broken in 5.7